backend/api/Controllers/DocumentController.cs

```diff
@@ -59,7 +59,7 @@ public DocumentController(IDocumentService documentService, IDocumentSyncService
         [SwaggerOperation(Tags = new[] { "documents" })]
         public IActionResult GetDocumentTypes()
         {
-            var result = _documentService.GetDocumentTypes();
+            var result = _documentService.GetStorageDocumentTypes();
```

Comment:
at some point we'll need claims for document crud. I don't think we called that out specifically with any of these stories but if it doesn't get done with these stories we may need a follow-up

Reply:
I agree. I prefer would prefer a story to go over the claims

Reply:
would you mind logging something so that we don't forget?